packages/design-system/src/components/ButtonAsLink/Primitive/ButtonPrimitiveAsLink.module.scss

```diff
@@ -0,0 +1,15 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+.button {
+	text-decoration: none;
+
+	&:hover,
+	&:focus {
+		text-decoration: none;
+	}
+
+	// External icon offset override in flex context
+	[data-test='link.icon.external'] {
```

Comment:
Remember those negative bottom margins on `Linkable` from earlier? Yeah, that's where they bite.

It's due to the nature of Link (inline) vs ButtonAsLink (inline-flex). IMO that's the least sucky way to do it.